src/main/js/store/actions/provider.js

```diff
@@ -0,0 +1,157 @@
+import { CONFIG_FETCHED, CONFIG_FETCHING, CONFIG_TEST_FAILED, CONFIG_TEST_SUCCESS, CONFIG_TESTING, CONFIG_UPDATE_ERROR, CONFIG_UPDATED, CONFIG_UPDATING } from 'store/actions/types';
```

Comment:
We should rename this from provider.js this is really just configuration methods across providers and channels. Perhaps configuration.js